src/index.js

```diff
@@ -136,18 +138,25 @@ const generateRuleText = (rule, aliases) => {
  *
  * Examples:
  *
- *     c6                       -> c6
- *     f captures g4 en passant -> fxg3
- *     bishop a takes e4        -> Baxe4
- *     castle queenside         -> O-O-O
- *     white resigns            -> 0-1
+ *     c6                       -> c6       ES: c6
+ *     f captures g4 en passant -> fxg3     ES: f captura g4 al paso
+ *     bishop a takes e4        -> Baxe4    ES: alfil por/toma e4
+ *     castle queenside         -> O-O-O    ES: enroque largo
+ *     white resigns            -> 0-1      ES: blancas abandonan
  */
 const ChessNLP = class ChessNLP {
 
-    constructor({ aliases } = { aliases: {} }) {
+    constructor({ aliases = {}, language = 'en' }) {
```

Comment:
This causes an exception if the constructor is called with no arguments, e.g.

```
import ChessNLP from 'chess-nlp';

const parser = new ChessNLP();
// TypeError: Cannot read property 'aliases' of undefined
```

Can be fixed like this:

```
    constructor({ aliases = {}, language = 'en' } = {}) {
```